data/transactions/logic/TEAL_opcodes.md

```diff
@@ -851,3 +861,19 @@ pushbytes args are not added to the bytecblock during assembly processes
 - LogicSigVersion >= 3
 
 pushint args are not added to the intcblock during assembly processes
+
+## callsub target
+
+- Opcode: 0x88
+- Pops: _None_
+- Pushes: _None_
+- branch unconditionally to TARGET, saving the next instruction on the call stack
```

Comment:
I think that we should document better that this stack is separated from the user-accessible stack.

Reply:
Added.